tests/cisco_ios/show_power_inline/cisco_ios_show_power_inline.yml

```
@@ -0,0 +1,170 @@
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/1"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/2"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/3"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/4"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/5"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/6"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/7"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/8"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/9"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "4"
+  Device: "C9120AXE-E"
+  Interface: "Gi1/0/10"
+  Max: "30.0"
+  Oper: "on"
+  Power: "23.2"
+- Admin: "auto"
+  Class: "4"
+  Device: "AIR-AP1562I-E-K9"
+  Interface: "Gi1/0/11"
+  Max: "30.0"
+  Oper: "on"
+  Power: "30.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/12"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/13"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/14"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/15"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/16"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/17"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/18"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/19"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/20"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "off"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/21"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/22"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/23"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/24"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
```

```suggestion
parsed_sample:
- admin_status: "auto"
  class: "4"
  device: "AIR-AP1562I-E-K9"
  interface: "Gi1/0/1"
  max: "30.0"
  operational_status: "on"
  power: "30.0"
- admin_status: "auto"
  class: "4"
  device: "AIR-AP1562I-E-K9"
  interface: "Gi1/0/2"
  max: "30.0"
  operational_status: "on"
  power: "30.0"
- admin_status: "auto"
  class: "4"
  device: "AIR-AP1562I-E-K9"
  interface: "Gi1/0/3"
  max: "30.0"
  operational_status: "on"
  power: "30.0"
- admin_status: "auto"
  class: "4"
  device: "C9120AXE-E"
  interface: "Gi1/0/10"
  max: "30.0"
  operational_status: "on"
  power: "23.2"
- admin_status: "auto"
  class: "4"
  device: "AIR-AP1562I-E-K9"
  interface: "Gi1/0/11"
  max: "30.0"
  operational_status: "on"
  power: "30.0"
- admin_status: "auto"
  class: "n/a"
  device: "n/a"
  interface: "Gi1/0/12"
  max: "30.0"
  operational_status: "off"
  power: "0.0"
- admin_status: "auto"
  class: "n/a"
  device: "n/a"
  interface: "Gi1/0/13"
  max: "30.0"
  operational_status: "off"
  power: "0.0"
```

ntc-templates will convert all capture groups to lowercase